assets/js/googlesitekit/datastore/ui/ui.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * `core/ui` data store: ui data
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import isPlainObject from 'lodash/isPlainObject';
+
+const SET_UI_VALUES = 'SET_UI_VALUES';
+const SET_UI_VALUE = 'SET_UI_VALUE';
+
+export const initialState = {};
+
+export const actions = {
+	/**
+	 * Stores site ui information.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {Object} values Ui data supplied as a map of keys to set and their respective values.
+	 * @return {Object} Redux-style action.
+	 */
+	setValues( values ) {
+		invariant( isPlainObject( values ), 'values must be an object.' );
+
+		return {
+			payload: { values },
+			type: SET_UI_VALUES,
+		};
+	},
+
+	/**
+	 * Sets a particular value for a key.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {string} key   Ui key to set the value for.
+	 * @param {string} value The value for the key.
+	 * @return {Object} Redux-style action.
+	 */
+	setValue( key, value ) {
+		invariant( key, 'key is required.' );
+		invariant( value, 'value is required.' );
```

Comment:
I don't think this is accurate either: `''`, `0`, `false`, etc. would all trigger this to fail which shouldn't happen.

I might not bother to check the `value` at all—even setting `undefined` here doesn't strike me as that terrible. But at most I'd check 
```suggestion
		invariant( value !== undefined, 'value is required.' );
```

Reply:
@tofumatt I think you're right. `value` can be anything which might not worth checking for. I've removed the test for it as well.